ibm_catalog.json

```diff
@@ -1181,7 +1182,7 @@
               "key": "enable_platform_metrics",
               "type": "string",
```

Comment:
@Ak-sky - This too is on the same lines as mentioned above for changing it back to the boolean type.